README.md

```diff
@@ -61,3 +61,40 @@ As mentioned above, package specifications are versioned. Versions follow the [s
 ## Version Compatibility between Packages and Specifications
 
 A package specifying its `format_version` as `x.y.z` must be valid against specifications in the semantic version range `[x.y.z, X.0.0)`, where `X = x + 1`.
+
+
+# What is an Elastic Package?
+
+An Elastic Package is a collection of assets for the Elastic Stack. In addition, it contains manifest files which contain additional information about the package. The exact content and structure of a package are described by the package spec. In the following is a high level overview of a package.
+
+A package is downloaded as a .zip file for the package-registry by Fleet inside Kibana. The zip file is then unpacked and each asset is installed into the related API.
```

Comment:
```suggestion
A package is downloaded as a .zip file from the package-registry by Fleet inside Kibana. The zip file is then unpacked and each asset is installed into the related API.
```

Reply:
I think we can leave out the `as a .zip file` here. Feels like an implementation detail.

Reply:
Hm, not sure. I often explain a package as a glorified zip file.

Reply:
What I mean is that the fact that a package is "downloaded as a .zip file" is entirely incidental as far as a package author or package consumer (end-user) is concerned. If I'm either of these personas, why should I care how a package is encoded in transport between the package registry and Kibana? Also tomorrow if we decide to deliver packages as `.tar` files or something else, are we going to remember to update this location here?